src/site/blocks/alert.drupal.liquid

```diff
@@ -35,38 +35,13 @@
       <p>{{ alert.fieldHelpText }}</p>
     {% endif %}
     {% if expander.fieldTextExpander %}
-      <button id="{{ expander.entityBundle }}-{{ expander.entityId }}" class="va-button-link" aria-expanded="false">{{ expander.fieldTextExpander }}</button>
-    {% endif %}
-    {% if expander.fieldTextExpander %}
-      <div id="{{ expander.entityBundle }}-{{ expander.entityId }}-content" class="expander-content expander-content-closed" aria-hidden="true">
-        <div class="expander-content-inner usa-alert-text">{{ expander.fieldWysiwyg.processed }}</div>
+      <div id="{{ expander.entityBundle }}-{{ expander.entityId }}" class="form-expanding-group borderless-alert additional-info-container" aria-hidden="true">
+        <div class="additional-info-title">{{ expander.fieldTextExpander }}</div>
+
+        {% if expander.fieldWysiwyg %}
+          <div class="additional-info-content usa-alert-text">{{ expander.fieldWysiwyg.processed }}</div>
+        {% endif %}
       </div>
     {% endif %}
-    {% if expander.fieldWysiwyg && expander.fieldTextExpander === empty %}
-      <div class=" usa-alert-text">{{ expander.fieldWysiwyg.processed }}</div>
-    {% endif %}
   </div>
 </div>
-
```

Comment:
@ncksllvn I refactored this template to utilize logic already being used in `createdAdditionalInfoWidget`. This definitely works for the alerts that have the additional info toggle functionality, I just need to make sure this doesn't break anything outside of that in the process. 

I like how this puts the responsibility of this toggling behavior all inside that single file. If you like this approach, we should comment in some clarification for the behavior that it controls, since right now it's completely implicit and not discernible from the template alone.

Reply:
This is awesome 💥 A design improvement while consolidating code.

I think the best test cases would be -

1. `/health-care/` - This would cover a Drupal page with a single alert.
2. `/careers-employment/` - This would cover a Markdown page with a single alert.
3. `/health-care/health-needs-conditions/mental-health/` - This would cover a Drupal page with more than one alert.

I don't think there are any Markdown pages with more than one alert.

I think you may already know this, but on your local machine, when you start the site, you should see some output like `Overriding conflicting vagov-content file: health-care/family-caregiver-benefits/champva.md`. This is an indicator that Drupal is overriding a Markdown page. If you open your console on one of those pages, you should see there `Metalsmith template generated from content: Object`  message to be extra sure it's from Drupal.

I think an engineer should be able to trace back the behavior for additional info, but [comments](https://shopify.github.io/liquid/tags/comment/) always help (when we update them and don't just leave them floating around for ages 😆 )

Reply:
yup! I was taking advantage of the Metalsmith object while debugging this one. Ok, I'll just do another sweep over those files while doing the utility classes to make sure everything looks ok

Reply:
everything looks good now and tests are passing! 🎉 I added a comment to add clarification around the implicit functionality, but should be good to merge